rule/rule.go

```diff
@@ -622,7 +625,8 @@ func addFilter(rule *ruleData, lhs, comparator, rhs string) error {
 		switch field {
 		case pidField, uidField, gidField, auidField, msgTypeField,
 			subjectUserField, subjectRoleField, subjectTypeField,
-			subjectSensitivityField, subjectClearanceField:
+			subjectSensitivityField, subjectClearanceField,
+			exeField, pathField, dirField:
```

Comment:
It would be nice to have a provenance for this list to extend beyond `msgTypeField`.

Reply:
Added. And I found that I was making a mistake here. 👍